src/index.html

```diff
@@ -1,13 +1,375 @@
+                <img
+                  src="images/photos/gallery6.jpg"
+                  alt="location image"
+                  class="location__image"
+                >
+              </div>
+            </div>
+            <div class="location__description">
+              <h3>Audio sunglasses</h3>
+            </div>
+          </div>
+
+          <a
+            href="#"
+            class="button button--view"
+          >
+            View all
+          </a>
+        </div>
+      </section>
+
+      <section class="page__section about" id="about-us">
+        <div class="container">
+          <div class="about__content">
+            <h2 class="about__title ">
+              Why buy direct
+              <br>
+              from Bose
+            </h2>
+            <div class="about__description">
+              <div class="about__subtitle">
+                Free 2-day shipping and returns
+              </div>
+              <div class="about__subtitle">
+                90-day risk-free trial
+              </div>
+              <div class="about__subtitle">
+                World class customer service
+              </div>
+              <div class="about__subtitle">
+                My Bose account management
+              </div>
+
+              <p class="about__paragraph">
+                A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+              </p>
+
+            </div>
+          </div>
+
+        </div>
+      </section>
+
+      <section class="page__section contacts" id="contacts">
+        <div class="container">
+          <h2 class="page__section--title">
+            Contact us
+          </h2>
+
+          <div class="contacts__content">
+            <div class="contacts__content__left">
+              <form
+                action="#"
+                method="GET"
+                class="form"
+                id="form"
+              >
+                <input
+                  class="form-field contacts__input"
+                  type="text"
+                  name="name"
+                  id="name"
+                  placeholder="Name"
+                  required
+                 >
+                <input
+                  type="email"
+                  class="form-field contacts__input"
+                  name="email"
+                  id="email"
+                  placeholder="Email"
+                  required
+                >
+                <textarea
+                  class="form-field form-field--message contacts__input contacts__input-last"
+                  name="message"
+                  placeholder="Message"
+                  id="message"
+                  required
+                ></textarea>
+                <button class="button button--primary">
+                  Send
```

Comment:
send button should be same length as form fields in all modes but here only mobile mode has proper width rest 2 modes do not
<img width="322" alt="image" src="https://user-images.githubusercontent.com/100297341/194698422-681ee94e-3db2-4c11-861b-72e560ddf9ba.png">